### PR TITLE
Extract support for MultipartFile to separate module.

### PR DESCRIPTION
**Pull request by tjuchniewicz**
Fixes gh-3.

### CODE REVIEW

File: feign-form-spring/pom.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <groupId>io.github.openfeign.form</groupId>
+        <artifactId>feign-form-parent</artifactId>
+        <version>2.0.1</version>
+    </parent>
+    
+    <name>Open Feign Forms Extension for Spring</name>   
+    
+    <dependencies>
+        <dependency>
+            <groupId>ru.xxlabaza</groupId>
```
Comment (xxlabaza) [inline feign-form-spring/pom.xml]: I think it should be replaced with:
```xml
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>feign-form</artifactId>
            <version>${project.version}</version>
            <scope>compile</scope>
        </dependency>
```

Comment (xxlabaza) [inline feign-form-spring/pom.xml]: 1) ru.xxlabaza group id doesn't exists any more
2) you have always current project version

Comment (tjuchniewicz) [inline feign-form-spring/pom.xml]: You are right! I'll fix this.

Comment (tjuchniewicz) [inline feign-form-spring/pom.xml]: Fixed.

**Pull request merged**